client_session_key.aes

```diff
@@ -0,0 +1 @@
+#�I̋9WD�{��3b��8�FU�q�r���@���ɘnD�I�i���]��6#�ᥟJ��U�w�E4�b?n�ɰp
�Y7CƦб�qK��
```

Comment:
What does this file do?

Reply:
It gets checked in by accident I think :). I believe it's part of the SecureCookie implementation via Web.ClientSession.